java/runtime/src/main/resources/ray.default.conf

```diff
@@ -60,9 +60,9 @@ ray {
     // If `redis.server` isn't provided, which port we should use to start redis server.
     head-port: 6379
     // The password used to start the redis server on the head node.
-    head-password: ""
+    head-password: "5241590000000000"
     // The password used to connect to the redis server.
-    password:""
+    password: "5241590000000000"
```

Comment:
add a comment that these above passwords should be consistent with the ones defined in python/ray/ray_constants.py

Reply:
Done.